certbot/compat.py

```diff
@@ -199,3 +199,19 @@ def get_default_folder(folder_type):
         return LINUX_DEFAULT_FOLDERS[folder_type]
     # Windows specific
     return WINDOWS_DEFAULT_FOLDERS[folder_type]
+
+
+def normalize_path(path):
+    # type: (str) -> str
+    """
+    Replace unsupported characters in path for current OS by underscores.
+    :param path: the path to normalize
+    :return: the normalized path
+    """
+    if 'fcntl' in sys.modules:
```

Comment:
Can we use `os.name` here?

Reply:
Done. I took the occasion to refactor a little `compat` module to be more consistent.